### PR TITLE
Handling PSEOF exception

### PR DESCRIPTION
**Pull request by jochman**
fixes https://github.com/demisto/etc/issues/13757

### CODE REVIEW

File: Scripts/script-ReadPDFFile.yml [hunk 1]
```diff
@@ -4,12 +4,33 @@ commonfields:
...
 script: |-
   import pdfx
+  from pdfminer.psparser import PSEOF
   import sys
   reload(sys)
   sys.setdefaultencoding('utf-8')
 
   entryID = demisto.args()['entryID']
 
+  def mark():
```
Comment (BenJoParadise) [inline Scripts/script-ReadPDFFile.yml]: The name can be more descriptive, for example:
```suggestion
  def mark_suspicious():
```

File: Scripts/script-ReadPDFFile.yml [hunk 2]
```diff
@@ -29,9 +50,18 @@ script: |-
   else:
       path = demisto.get(res[0],'Contents.path')
       if path:
-          pdf = pdfx.PDFx(path)
+          try:
+              pdf = pdfx.PDFx(path)
+          except PSEOF:
+              mark()
```
Comment (BenJoParadise) [inline Scripts/script-ReadPDFFile.yml]: Same as above comment
```suggestion
              mark_suspicious()
```

**Comment by BenJoParadise** [comment]
Also, you should add a test for this, please consult @ronykoz if you need help doing this.

**Comment by ronykoz** [comment]
@jochman you should add the pdf file in another PR, otherwise your test won't be able to load the file from git, as it is not there yet. Afterwards you need to add the TestPlaybook for this script.
And I also want you to add a regular PDF and make sure it is not marked as a suspicious one(Another check for the TestPlaybook).

**Pull request merged**